src/js/core/methods/CardanoSignTransaction.js

```diff
@@ -242,25 +233,16 @@ export default class CardanoSignTransaction extends AbstractMethod<'cardanoSignT
     _ensureFirmwareSupportsParams() {
         const { params } = this;
 
-        params.certificatesWithPoolOwnersAndRelays.forEach(({ certificate }) => {
-            if (certificate.type === CardanoCertificateType.STAKE_POOL_REGISTRATION) {
-                this._ensureFeatureIsSupported('SignStakePoolRegistrationAsOwner');
-            }
+        // we no longer support non-streamed signing
+        this._ensureFeatureIsSupported('TransactionStreaming');
```

Comment:
Maybe we could change the min supported version for `cardanoSignTransaction` in `src/data/config.json`? We wouldn't have to do the check here and the general Connect "New firmware update available" screen would be shown instead of a "random" failure.
![image](https://user-images.githubusercontent.com/7520735/164002280-9bdc1a5f-5e63-4a69-a403-fa4e59401d27.png)

Reply:
Ah great, I missed that, thanks!